### ISSUE TITLE
Merge Summaries concept into Topics and Popular

### ISSUE THREAD

**Issue by georgeowell** [fraction/oasis#249]
## What's the problem you want solved?
Summaries is pretty decent now. I just wondered if anyone wanted to merge that visible comments concept into Topics and Popular. 

This would be a much better experience in navigating public threads as it allow you to see that a post has become a discussion, a preview of the nature of discussion and also how many comments exist. 

In addition, when combined with #179, we would have a clearer label to click through to the thread rather than clicking a unrelated piece of information (how long since the post was made).

I also wish the visible comments were limited to two rather than three as it takes up quite a bit of screen space on my laptop.

## Is there a solution you'd like to recommend?
- Removing summaries view
- Adding comment summaries to Topics view
- Adding comment summaries to Popular view

**Comment by christianbundy** [comment]
I'd merge this but I think it would be more prudent to make filters or something like in #170 so that we can switch between the raw view and the summary view everywhere.

**Comment by georgeowell** [comment]
> I'd merge this but I think it would be more prudent to make filters or something like in #170 so that we can switch between the raw view and the summary view everywhere.

Fair enough! Perhaps https://github.com/fraction/oasis/issues/179 would be a good step for now then?

**Comment by georgeowell** [comment]
I think perhaps Topics and Summaries should just be merged into Threads. Threads seem like a good compromise for what those views were trying achieve.
